lib/regions.js

```diff
@@ -33,106 +30,180 @@ const update_region = query.bind(query, fs.readFileSync('./sql/update_region.sql
 }
 
-async function update(pg_pool, name, country, description, locale, private_capable, provider_name, provider_region, provider_availability_zones, high_availability, deprecated) {
+async function update(
+  pg_pool,
+  name,
+  country,
+  description,
+  locale,
+  private_capable,
+  provider_name,
+  provider_region,
+  provider_availability_zones,
+  high_availability,
+  deprecated,
+) {
   try {
-    assert.ok(!country || country !== '', 'The specified country field does not exist and is required.')
-    assert.ok(!description || description !== '', 'The description field for this region is required.')
-    assert.ok(!locale || locale !== '', 'The specified locale must exist.')
-    assert.ok(!provider_name || provider_name !== '', 'The specified providers name must exist.')
-    assert.ok(!provider_region || provider_region !== '', 'The specified providers region must exist.')
-    assert.ok(!provider_availability_zones || Array.isArray(provider_availability_zones), 'The specified providers availability_zones is required.')
+    assert.ok(!country || country !== '', 'The specified country field does not exist and is required.');
+    assert.ok(!description || description !== '', 'The description field for this region is required.');
+    assert.ok(!locale || locale !== '', 'The specified locale must exist.');
+    assert.ok(!provider_name || provider_name !== '', 'The specified providers name must exist.');
+    assert.ok(!provider_region || provider_region !== '', 'The specified providers region must exist.');
+    assert.ok(!provider_availability_zones || Array.isArray(provider_availability_zones), 'The specified providers availability_zones is required.');
   } catch (e) {
-    throw new common.UnprocessibleEntityError(e.message)
+    throw new common.UnprocessibleEntityError(e.message);
   }
-  let updated_region = await update_region(pg_pool, [name, country, description, locale, private_capable, provider_name, provider_region, provider_availability_zones ? provider_availability_zones.join(',') : null, high_availability, deprecated])
-  if(updated_region.length !== 1) {
-    throw new common.InternalServerError('An unexpected error occured updating this regions record.')
+  const updated_region = await update_region(
+    pg_pool,
+    [
+      name,
+      country,
+      description,
+      locale,
+      private_capable,
+      provider_name,
+      provider_region,
+      provider_availability_zones ? provider_availability_zones.join(',') : null,
+      high_availability,
+      deprecated,
+    ],
+  );
+  if (updated_region.length !== 1) {
+    throw new common.InternalServerError('An unexpected error occured updating this regions record.');
   }
-  return await updated_region[0]
+  return updated_region[0];
 }
 
-async function create(pg_pool, name, country, description, locale, private_capable, provider_name, provider_region, provider_availability_zones, high_availability) {
+async function create(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Async function 'create' has a complexity of 12. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5244943)